src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  heigth?: number;
+  width?: number;
```

Comment:
Згідно умови, в інтерфейсф достатньо прописати shape, color & method!!!

Write an interface `Figure` and 3 classes implementing it so that every figure 
has:
- a `shape` (`triangle`, `circle` or `rectangle`);
- a `color` (`red`, `green` or `blue`);
- a method `getArea` that returns the area of the figure rounded down to 
hundredths.

Reply:
Згоден, але мене здається, що можна і залишити, тоді можна уникнути помилок через неправильне написання. Наприклад, якщо хтось напише height замість heigth далі у коді. Але, скоріше, це дійсно зайве.